listing.php

```diff
@@ -333,15 +334,28 @@ function showTreeDir($svnrep, $path, $rev, $peg, $listing) {
 	$vars['compare_endform'] = '</form>';
 
 	$vars['showlastmod'] = $config->showLastModInListing();
-
-	$listing = showTreeDir($svnrep, $path, $rev, $peg, array());
-
+	//$listing = showTreeDir($svnrep, $path, $rev, $peg, array());
+	$listing = showTreeDir($svnrep, $path, $rev, $peg, $listing,$index);
```

Comment:
Same here.

Reply:
Removed as these were the changes for #104